### PR TITLE
Fixed some typos in "05_Sampling_potential_energy_surface"

### PR DESCRIPTION
**Pull request by divshacker**
### Summary
This PR is clearly the fix of #1124 . There were  some typos in [05_Sampling_potential_energy_surface](https://qiskit.org/documentation/tutorials/chemistry/05_Sampling_potential_energy_surfaces.html)
in Extrapolation Section.

### Details and comments
I have tried to fix the problem please have a look and let me know if any changes required .
Thank you

### CODE REVIEW

**Comment by divshacker** [comment]
@SooluThomas @nonhermitian @Feigenbaum4669 Please have a look and let me know

**Comment by divshacker** [comment]
I don't know why all test got failed. Give me some time . 
I will push it again.

**Comment by divshacker** [comment]
@SooluThomas @nonhermitian Please check now I think the problem is fixed now . 
Thank You

**Comment by divshacker** [comment]
@SooluThomas are you guys doing that ?

**Comment by divshacker** [comment]
@SooluThomas  please review this pr also and let me know any changes required.  It's been two weeks now .😅🙂

File: tutorials/chemistry/05_Sampling_potential_energy_surfaces.ipynb [hunk 1]
```diff
@@ -1,534 +1,609 @@
...
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "seAv2GOtU_-n"
+      },
+      "source": [
+        "Here, we use the H2 molecule as a model sysem for testing."
```
Comment (SooluThomas) [inline tutorials/chemistry/05_Sampling_potential_energy_surfaces.ipynb:95]:
```suggestion
        "Here, we use the H2 molecule as a model system for testing."
```

**Review by SooluThomas — COMMENTED** [review]
Adding some minor suggestions here

File: tutorials/chemistry/05_Sampling_potential_energy_surfaces.ipynb [hunk 2]
```diff
@@ -1,534 +1,609 @@
 {
```
Comment (SooluThomas) [inline tutorials/chemistry/05_Sampling_potential_energy_surfaces.ipynb:1]: <p>Can you please re-run this cell with the most recent versions?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (divshacker) [reply tutorials/chemistry/05_Sampling_potential_energy_surfaces.ipynb:1]: Ok I will.

**Comment by divshacker** [comment]
Hey @SooluThomas Sorry for responding Late, I was busy with college stuff.
I think I have fixed all as you have suggested.
Please have a look and let me know.
Thank You

**Comment by divshacker** [comment]
Hey @SooluThomas I don't know how the test got Failed. Every thing looks Good to ne, Can you please try to fix that?
Thank You

**Comment by divshacker** [comment]
Hey @nonhermitian Can you please Review this PR and let me know any changes needed. 
Thank You

**Comment by divshacker** [comment]
@nonhermitian

**Comment by divshacker** [comment]
Hey @SooluThomas  Can you please check this PR too.

**Review by mtreinish — APPROVED** [review]

**Comment by divshacker** [comment]
Hey @SooluThomas @nonhermitian I don't know why it still not merging automatically. Can you restart the test as it will merge after successful completion of all test.

**Pull request merged**